crypto/Kconfig

```diff
@@ -3,6 +3,13 @@ menu "Crypto libraries for nRF5x SOCs."
 config NRFXLIB_CRYPTO
 	bool
 
+config SHORT_WCHAR_T
+	bool "Compile with short wchar_t size (2 bytes)"
+	default n
```

Comment:
default n is implied and should be omitted.

Reply:
Fixed